### PR TITLE
Remove code related to the PaaS

### PR DESCRIPTION
**Pull request by thomasleese**
This removes code related to the PaaS as we've now completed the migration to AKS. I've removed the service unavailable page as they currently won't work, although we may need to bring them back in the future.

[Trello Card](https://trello.com/c/TjTM7Sxi/2087-switch-off-old-paas-infrastructure)

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -52,13 +52,7 @@ jobs:
         run: terraform fmt -check -recursive
         working-directory: terraform
 
-      - name: Validate PaaS
-        run: |
-          terraform init -backend=false
-          terraform validate -no-color
-        working-directory: terraform/paas
-
-      - name: Validate AKS
```
Comment (richardpattinson) [inline .github/workflows/lint.yml:61]: Why have this name change?

Comment (thomasleese) [reply .github/workflows/lint.yml:61]: No particular reason other than we're only validating one thing now.

File: .gitignore [hunk 2]
```diff
@@ -10,7 +10,6 @@
 .env.production.local
 .env.test.local
 app/assets/builds/*
-bin/fetch_config.rb
```
[the remark this answers is not in the document]
Comment (thomasleese) [reply .gitignore:13]: We no longer use this script, it's been removed from the `Makefile` so we don't need to ignore it as it won't exist.

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1578/commits/8a553e521c73172990a757081f370bf03d4ba634#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L102-L105

File: bin/restore-db [hunk 3]
```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```
Comment (richardpattinson) [inline bin/restore-db:1]: What does this file do/ why do we no longer need it?

File: service_unavailable_page/web/mime.types [hunk 4]
```diff
@@ -1,78 +0,0 @@
-types {
```
Comment (richardpattinson) [inline service_unavailable_page/web/mime.types:1]: Again; im unsure what this mime types file is doing; something to do with the service unavailable page?

File: service_unavailable_page/web/nginx.conf [hunk 5]
```diff
@@ -1,35 +0,0 @@
-worker_processes 1;
```
Comment (richardpattinson) [inline service_unavailable_page/web/nginx.conf:1]: Same as above; what are these for?

File: bin/restore-db (hunk 3 above)
Comment (thomasleese) [reply bin/restore-db:1]: It used to be able to restore the database of a PaaS environment, so it'll no longer work on AKS. We should have an alternative, but that's for a separate PR.

File: service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg [hunk 6]
```diff
@@ -1,7 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="132" height="97" viewBox="0 0 132 97" version="1.1">
```
Comment (richardpattinson) [inline service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg:1]: Why does the service unavailable page create so many files?

File: service_unavailable_page/web/mime.types (hunk 4 above)
Comment (thomasleese) [reply service_unavailable_page/web/mime.types:1]: The service unavailable page was deployed to the PaaS, and therefore doesn't work on AKS. We probably do want an alternative for the future but I think it's confusing to keep code around which doesn't work, so until then we can remove it.

This particular file was a configuration file for Nginx, to make sure it served the files with the correct type for the browser.

File: service_unavailable_page/web/public/internal/index.html [hunk 7]
```diff
@@ -1,167 +0,0 @@
...
-      Status - GOV.UK
-    </title>
-    <meta name="robots" content="noindex, nofollow" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, viewport-fit=cover"
-    />
-    <meta name="theme-color" content="#0b0c0c" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <link
```
Comment (richardpattinson) [inline service_unavailable_page/web/public/internal/index.html:16]: Why does this page have so many images?

File: service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg (hunk 6 above)
Comment (thomasleese) [reply service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg:1]: The service unavailable page was designed to be deployed without any dependency on our service, as the service would be down, so it needed its own copy of the GOV.UK Design system.

File: service_unavailable_page/web/public/internal/index.html (hunk 7 above)
Comment (thomasleese) [reply service_unavailable_page/web/public/internal/index.html:16]: The service unavailable page was designed to be deployed without any dependency on our service, as the service would be down, so it needed its own copy of the GOV.UK Design system.

**Review by richardpattinson — COMMENTED** [review]
Would like some clarity on some comments about files we are removing; Cannot see any issues otherwise. Once the comments are addressed I will approve

File: service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg (hunk 6 above)
Comment (richardpattinson) [reply service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg:1]: Makes a lot of sense. cheers for the clarification

**Review by richardpattinson — APPROVED** [review]
Cheers for the clarification; This all seems like extraneous code now after the migration.

**Pull request merged**